doc/gtdb.md

```diff
@@ -0,0 +1,137 @@
+
+```
+RS_GCF_000008865.1	d__Bacteria;p__Proteobacteria;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Escherichia;s__Escherichia dysenteriae
+```
+
+This format can be directly parsed by Woltka:
+
+```bash
+woltka classify --lineage bac120_taxonomy_r89.tsv ...
+```
+
+If you work with both Bacteria and Archaea (this is the common case), you may simply concatenate the two taxonomy files before use:
+
+```bash
+cat bac120_taxonomy_r89.tsv ar122_taxonomy_r89.tsv > gtdb_taxonomy.tsv
+```
+
+Note: The [GTDB-Tk](https://github.com/Ecogenomics/GtdbTk) database ([`gtdbtk_data.tar.gz`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/gtdbtk_r89_data.tar.gz)) already provides a combined `gtdb_taxonomy.tsv` file.
+
+
+## Reformat GTDB as taxdump style
+
+Alternatively, you may consider converting GTDB taxonomy into the format of NCBI taxdump. This will allow you to connect GTDB to more bioinformatics tools such as BLAST and Kraken2 which favor the taxdump format. This will also make the Woltka output slimer, if that's your preference. For example, instead of the long lineage strings like `d__Bacteria;p__Proteobacteria;...`, you will get taxon names like `Escherichia` as the feature name.
+
+Three good things of the GTDB taxonomy (in addition to many others) are that:
+
+1) All lineages have exactly seven ranks (`d`omain, `p`hylum, `c`lass, `o`rder, `f`amily, `g`enus and `s`pecies).
+2) All ranks are filled (i.e., no "gaps" such as `g__;` as multiple other systems do).
+3) At each rank, all taxon names are unique (but duplicate names are allowed at different ranks).
+
+These characteristics make it safe to collapse GTDB lineages into taxonomic units and index them in a way that resembles NCBI taxdump (I mean, in case you are thinking about whether this trick also applies to other taxonomies).
+
+We provide a script: [`gtdb_to_taxdump.py`](https://biocore.github.io/wol/code/scripts/gtdb_to_taxdump.py) to automate this conversion:
+
+```bash
+gtdb_to_taxdump.py bac120_taxonomy_r89.tsv ar122_taxonomy_r89.tsv
+```
+
+If you start with `gtdb_taxonomy.tsv` from [GTDB-Tk](https://github.com/Ecogenomics/GtdbTk), do this instead:
+
+```bash
+gtdb_to_taxdump.py gtdb_taxonomy.tsv
+```
+
+This command will generate three files: `nodes.dmp`, `names.dmp` and `taxid.map`. You can probably already guess what's in them. But to make it explicit:
+
+Every taxon is given a taxonomy ID. This ID is an incremental integer, starting from "1" at the root, and increasing from higher ranks to lower ranks. So domains Bacteria and Archaea receive "2" and "3", respectively, followed by phyla, so on so forth.
```

Comment:
Just from reading this description, I'm not sure what is in nodes.dmp and how it relates to names.dmp. I do understand that taxid.map is a mapping from taxon to and ID

Reply:
nodes.dmp and names.dmp are the NCBI-style taxonomy database files. See [here](ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz) for the official version. I reworded this a bit.